core/jobs_test.py

```diff
@@ -832,13 +983,52 @@ def get_count(cls, exploration_id):
         return answer
 
 
+TIMES_RUN = 0
+
+
+class MockMRJobManager(jobs.BaseMapReduceJobManagerForContinuousComputations):
+
+    @classmethod
+    def _get_continuous_computation_class(cls):
+        return MockContinuousComputationManager
+
+    @classmethod
+    def entity_classes_to_map_over(cls):
+        return []
+
+
+class MockContinuousComputationManager(jobs.BaseContinuousComputationManager):
+
+    @classmethod
+    def get_event_types_listened_to(cls):
+        return []
+
+    @classmethod
+    def _get_realtime_datastore_class(cls):
+        return MockStartExplorationRealtimeModel
+
+    @classmethod
+    def _get_batch_job_manager_class(cls):
+        return MockMRJobManager
+
+    @classmethod
+    def _kickoff_batch_job_after_previous_one_ends(cls):
+        global TIMES_RUN  # pylint: disable=global-statement
+        if TIMES_RUN < 2:
+            (
```

Comment:
Ditto, I don't think you need the outer parens.

Reply:
cant drop parens